.github/renovate.json5

```diff
@@ -1,9 +1,8 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:recommended",
```

Comment:
Why remove this?

Reply:
it's replaced by `config:best-practices`, check out https://docs.renovatebot.com/upgrade-best-practices/